packages/frontend/src/components/dialogs/FullscreenMedia.tsx

```diff
@@ -60,6 +60,7 @@ export default function FullscreenMedia(props: Props & DialogProps) {
     previous: false,
     next: false,
   })
+  const [contextMenuOpen, setContextMenuOpen] = useState(false)
```

Comment:
This never gets reset to `false`, resulting in `contextMenuOpen` being incorrectly `true` when the context menu is actually closed. Although currently the only downside to this is that you have to press "Escape" twice to close the full screen dialog.

I think this hook should be used instead:

https://github.com/deltachat/deltachat-desktop/blob/cc4fd19cd64a32ca8edca2151ef1b4272cf231e1/packages/frontend/src/components/ContextMenu.tsx#L455-L469